src/server/src/main/resources/db/h2/V3_0_0__table_blacklist.sql

```diff
@@ -3,4 +3,4 @@
 --
 
 ALTER TABLE repair_unit 
-ADD blacklisted_tables ARRAY NOT NULL;
\ No newline at end of file
+ADD blacklisted_tables ARRAY;
```

Comment:
nit: this should be fixed in the previous commit (every sha in the PR should build and test ok, otherwise we break eg `git bisect`)